network/local.go

```diff
@@ -7,17 +7,14 @@ import (
 	"time"
 )
 
-// NewLocalRouter returns a fresh router which uses only local queues. It uses
-// the default local manager.
-// If you need multiple independent local-queues, use NewLocalRouterWithManager.
-// In case of an error it is returned together with a nil-Router.
+// NewLocalRouter returns a fresh router which uses local connections. It takes
```

Comment:
There are also some other changes we did in 3/7 - a short overview of the file in `mocking_local` looks good, I'll copy that to here. Except:

```
// 127.0.0.1:2000 -> 127.0.0.1:7869 => 14
// 127.0.0.1:7869 <- 127.0.0.1:2000 => 15
```

Have a good look at it and tell me what's wrong ;)